pkgs/by-name/cd/cd2netmd-gui/package.nix

```
@@ -0,0 +1,150 @@
+{
+  atracdenc,
+  bash,
+  cmake,
+  copyDesktopItems,
+  fetchFromGitHub,
+  ffmpeg,
+  lib,
+  libcdio,
+  libcdio-paranoia,
+  libgcrypt,
+  libgpg-error,
+  libusb1,
+  makeDesktopItem,
+  netmdplusplus,
+  nlohmann_json,
+  pkg-config,
+  qt5,
+  stdenv,
+  taglib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cd2netmd-gui";
+  version = "2.1.4";
+
+  src = fetchFromGitHub {
+    owner = "Jo2003";
+    repo = "cd2netmd_gui";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-WPcs96xA3FGo62pIKC+xwqm4s3Gxo0l9HMx8vmH+9xg=";
+  };
+
+  buildInputs = [
+    libcdio
+    libcdio-paranoia
+    libgcrypt
+    libgpg-error
+    libusb1
+    netmdplusplus
+    nlohmann_json
+    qt5.qtbase
+    taglib
+  ];
+
+  postPatch = ''
+    # Source uses git tags to determine version. We replace that here to avoid
+    # a build dependency on the .git directory
+    echo "#define GIT_VERSION \"v${finalAttrs.version}\"" > git_version.h
+    substituteInPlace CMakeLists.txt \
+      --replace-fail \
+        'add_custom_target(git_version_h COMMAND cd ${"$"}{CMAKE_SOURCE_DIR} && ./upd_git_version.sh)' \
+        ""
+    substituteInPlace CMakeLists.txt \
+      --replace-fail \
+        'add_dependencies(netmd_wizard git_version_h)' \
+        ""
+
+    # Fix CLI invocation of atracdenc to use absolute path
+    substituteInPlace cxenc.h \
+      --replace-fail \
+        'static constexpr const char* XENC_CLI = "toolchain/atracdenc.exe";' \
+        'static constexpr const char* XENC_CLI = "${lib.getExe atracdenc}";'
+    substituteInPlace cxenc.h \
+      --replace-fail \
+        'static constexpr const char* XENC_CLI = "atracdenc";' \
+        'static constexpr const char* XENC_CLI = "${lib.getExe atracdenc}";'
+
+    # Fix CLI invocation of ffmpeg to use absolute path
+    substituteInPlace cffmpeg.h \
+      --replace-fail \
+        'static constexpr const char* FFMPEG_CLI = "toolchain/ffmpeg.exe";' \
+        'static constexpr const char* FFMPEG_CLI = "${lib.getExe ffmpeg}";'
+    substituteInPlace cffmpeg.h \
+      --replace-fail \
```

multiple replace fails can be used for one substituteInPlace
```suggestion
    substituteInPlace cffmpeg.h \
      --replace-fail \
        'static constexpr const char* FFMPEG_CLI = "toolchain/ffmpeg.exe";' \
        'static constexpr const char* FFMPEG_CLI = "${lib.getExe ffmpeg}";' \
      --replace-fail \
```

---

Thank you. Consolidated `substituteInPlace` calls that operate on the same file set.